singularity-eos/eos/eos_base.hpp

```diff
@@ -80,7 +80,10 @@ char *StrCat(char *destination, const char *source) {
   using EosBase<EOSDERIVED>::FillEos;                                                    \
   using EosBase<EOSDERIVED>::EntropyFromDensityTemperature;                              \
   using EosBase<EOSDERIVED>::EntropyFromDensityInternalEnergy;                           \
-  using EosBase<EOSDERIVED>::EntropyIsNotEnabled;
+  using EosBase<EOSDERIVED>::EntropyIsNotEnabled;                                        \
```

Comment:
I assume these would be required @Yurlungur, is any of this machinery tested on unmodified Eos's?

Reply:
Just an FYI - I think you can remove this completely since it's automatically slurped in via inheritance

Reply:
I think @jhp-lanl is correct. I *think* the machinery is checked on unmodified EOS's explicitly. But even if it is not, the "pull out an unmodified EOS" gets automatically tested on unmodified EOS's because that's the base case of the recursion algorithm.